x-pack/plugins/security_solution/public/flyout/left/hooks/use_threat_intelligence_details.test.ts

```diff
@@ -76,7 +77,7 @@ describe('useThreatIntelligenceDetails', () => {
       },
       dataAsNestedObject: null,
       investigationFields: [],
-    });
+    } as unknown as LeftPanelContext);
```

Comment:
we needed the cast because some of the properties don't mach the interface, now that it doesn't accept null or undefined.
You're correct though, using `mockContextValue` is a lot easier, so I replaced it!